.github/CODEOWNERS

```diff
@@ -0,0 +1,473 @@
+/llvm/include/llvm/XRay/ @llvm/pr-subscribers-xray
+/llvm/lib/XRay/ @llvm/pr-subscribers-xray
+/llvm/tools/llvm-xray/ @llvm/pr-subscribers-xray
+/llvm/unittests/XRay/ @llvm/pr-subscribers-xray
+/compiler-rt/*/xray/ @llvm/pr-subscribers-xray
+
+# clang-codegen
+/clang/lib/CodeGen/** @llvm/pr-subscribers-clang-codegen
+/clang/include/clang/CodeGen/ @llvm/pr-subscribers-clang-codegen
+
+# mlir
+/mlir/ @llvm/pr-subscribers-mlir
+
+# mlir-core
+/mlir/**/IR/ @llvm/pr-subscribers-mlir-core
+/mlir/**/IR/ @llvm/pr-subscribers-mlir-core
+/mlir/**/Support/ @llvm/pr-subscribers-mlir-core
+/mlir/**/Parser/ @llvm/pr-subscribers-mlir-core
+/mlir/**/IR/ @llvm/pr-subscribers-mlir-core
+/mlir/**/Bytecode/ @llvm/pr-subscribers-mlir-core
+/mlir/**/AsmParser/ @llvm/pr-subscribers-mlir-core
+/mlir/**/Pass/ @llvm/pr-subscribers-mlir-core
+/mlir/**/tools/ @llvm/pr-subscribers-mlir-core
+/mlir/**/Reducer/ @llvm/pr-subscribers-mlir-core
+/mlir/**/Transforms/ @llvm/pr-subscribers-mlir-core
+/mlir/**/Debug/ @llvm/pr-subscribers-mlir-core
+/mlir/tools/ @llvm/pr-subscribers-mlir-core
+
+# mlir-ods
+/mlir/TableGen/ @llvm/pr-subscribers-mlir-ods
+/mlir/tblgen/ @llvm/pr-subscribers-mlir-ods
+
+# mlir-bindings
+/mlir/Bindings/ @llvm/pr-subscribers-mlir-bindings
+
+# mlir-gpu
+/mlir/GPU/ @llvm/pr-subscribers-mlir-gpu
+
+# mlir-amdgpu
+/mlir/**/AMDGPU/ @llvm/pr-subscribers-mlir-amdgpu
+
+# mlir-amx
+/mlir/**/AMX/ @llvm/pr-subscribers-mlir-amx
+
+# mlir-affine
+/mlir/**/Affine/ @llvm/pr-subscribers-mlir-affine
+
+# mlir-arith
+/mlir/**/Arith/ @llvm/pr-subscribers-mlir-arith
+
+# mlir-neon
+/mlir/**/ArmNeon/ @llvm/pr-subscribers-mlir-neon
+
+# mlir-sme
+/mlir/**/ArmSME/ @llvm/pr-subscribers-mlir-sme
+
+# mlir-sve
+/mlir/**/ArmSVE/ @llvm/pr-subscribers-mlir-sve
+
+# mlir-async
+/mlir/**/Async/ @llvm/pr-subscribers-mlir-async
+
+# mlir-bufferization
+/mlir/**/Bufferization/ @llvm/pr-subscribers-mlir-bufferization
+
+# mlir-complex
+/mlir/**/Complex/ @llvm/pr-subscribers-mlir-complex
+
+# mlir-cf
+/mlir/**/ControlFlow/ @llvm/pr-subscribers-mlir-cf
+
+# mlir-dlti
+/mlir/**/DLTI/ @llvm/pr-subscribers-mlir-dlti
+
+# mlir-emitc
+/mlir/**/EmitC/ @llvm/pr-subscribers-mlir-emitc
+
+# mlir-func
+/mlir/**/Func/ @llvm/pr-subscribers-mlir-func
+
+# mlir-async
+/mlir/GPU/ @llvm/pr-subscribers-mlir-async
+
+# mlir-irdl
+/mlir/**/IRDL/ @llvm/pr-subscribers-mlir-irdl
+
+# mlir-index
+/mlir/**/Index/ @llvm/pr-subscribers-mlir-index
+
+# mlir-llvm
+/mlir/**/LLVM/ @llvm/pr-subscribers-mlir-llvm
+
+# mlir-linalg
+/mlir/linalg/ @llvm/pr-subscribers-mlir-linalg
+
+# mlir-mlprogram
+/mlir/**/MLProgram/** @llvm/pr-subscribers-mlir-mlprogram
+
+# mlir-math
+/mlir/**/Math/ @llvm/pr-subscribers-mlir-math
+
+# mlir-memref
+/mlir/**/MemRef/ @llvm/pr-subscribers-mlir-memref
+
+# mlir-nvgpu
+/mlir/**/NVGPU/ @llvm/pr-subscribers-mlir-nvgpu
+
+# mlir-openacc
+/mlir/**/OpenACC/ @llvm/pr-subscribers-mlir-openacc
+
+# mlir-openmp
+/mlir/**/OpenMP/ @llvm/pr-subscribers-mlir-openmp
```

Comment:
If this can be `/mlir/**/*OpenMP*/` then it will cover the `OpenMPToLLVM` and `SCFToOpenMP` conversions.